<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addcar</class>
 <widget class="QDialog" name="addcar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>551</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Imagine.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="car_number_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;</string>
   </property>
   <property name="text">
    <string>Car NO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_but">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>500</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ravie</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:grey;
color:black;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_but">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ravie</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:green;
color:black;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="Add_car_title">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;</string>
   </property>
   <property name="text">
    <string>Add Car</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="car_number_lineedit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
color:black;
border:2px solid grey;</string>
   </property>
  </widget>
  <widget class="QLabel" name="brand_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;</string>
   </property>
   <property name="text">
    <string>Brand</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="brand_lineedit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
border:2px solid grey;
color:black;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="car_type_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;</string>
   </property>
   <property name="text">
    <string>Car type</string>
   </property>
  </widget>
  <widget class="QListWidget" name="car_type_listwidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
border:2px solid grey;
color:black;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="fare_label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;</string>
   </property>
   <property name="text">
    <string>Fare</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fare_line_edit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
border:2px solid grey;
color:black;
</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
